src/olympia/reviewers/management/commands/migrate_old_info_requests.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+
+import olympia.core.logger
+
+from olympia.addons.models import AddonReviewerFlags
+from olympia.versions.models import Version
+
+
+log = olympia.core.logger.getLogger('z.reviewers.migrate_old_info_requests')
+
+
+class Command(BaseCommand):
+    help = 'Notify developers with pending info requests about to expire'
+
+    def handle(self, *args, **options):
+        deadline = datetime.now()
+        qs = Version.objects.no_cache().raw(
+            'SELECT id, addon_id, version FROM versions '
+            'WHERE has_info_request = true '
+            'GROUP BY addon_id')
+        for version in qs:
+            addon = version.addon
+            log.info('Migrating flag for addon %d', addon.pk)
+            AddonReviewerFlags.objects.update_or_create(addon=addon, defaults={
```

Comment:
Does django fetch the instance there? Otherwise you should be able to just use addon_ID right?

Reply:
I was wondering that as well, but for a one-off command I don't care enough :)